packit/local_project.py

```diff
@@ -239,11 +241,12 @@ def _parse_namespace_from_git_project(self):
 
     def _parse_git_url_from_git_repo(self):
         if self.git_repo and not self.git_url:
-            # this is prone to errors
-            # also if we want url to upstream, we may want to ask for it explicitly
-            # since this can point to a fork
-            # .urls returns generator
-            self.git_url = list(self.git_repo.remote().urls)[0]
+            if self.remote:
+                self.git_url = list(self.git_repo.remote(self.remote).urls)[0]
```

Comment:
```suggestion
                self.git_url = next(self.git_repo.remote(self.remote).urls)
```
should be more efficient